test/integration/api/run.bats

```diff
@@ -99,6 +99,35 @@ function teardown() {
 	[[ "${output}" == *"someDnsOption"* ]]
 	# stop-signal
 	[[ "${output}" == *"\"StopSignal\": \"SIGKILL\""* ]]
+
+	# following options are introduced in docker 1.10, skip older version
+	run docker --version
+	if [[ "${output}" != "Docker version 1.1"* ]]; then
```

Comment:
This skips future versions like '1.20' or '2.x'. Maybe you can skip docker 1.9 which is the only version it doesn't support.